src/ngraph/descriptor/tensor.hpp

```diff
@@ -45,8 +45,12 @@ namespace ngraph
             Tensor(const element::Type& element_type,
                    const PartialShape& pshape,
                    const std::string& name);
+            Tensor(const element::Type& element_type,
+                   const PartialShape& pshape,
+                   Node* node,
+                   size_t node_output_number);
 
-            const std::string& get_name() const { return m_name; }
+            const std::string& get_name() const;
```

Comment:
With this change, is anything using `m_name` anymore?

Reply:
All the ops that don't start with 'A'.